content/committees/specification/tckprocess/index.md

```diff
@@ -23,6 +25,8 @@ This document defines:
 
 -   Process for self-certification
 
+-   Process for releasing a point revision of a TCK
```

Comment:
Are we going to differentiate between a minor point release (x.y) versus a service release (x.y.z)?  Or, will the process be the same?

Reply:
From a TCK versioning perspective, we always use x.y.z (e.g. 5.0.1 for Servlet TCK or 9.0.1 for Platform TCK).  We kind of know that and aren't questioning that.  Just mentioning for those following this discussion and wondering which version the x.y vs x.y.z refers to.

My intention with using the words that I did in `Process for releasing a point revision of a TCK` was to cover what we do when addressing TCK challenges with a TCK x.y.z release with the challenged tests made to pass via test exclusion (or other workaround to be mentioned).

Perhaps the intention would be clearer with different wording `Release a service release (x.y.z) to resolve a TCK challenge`.

> Are we going to differentiate between a minor point release (x.y) versus a service release (x.y.z)? Or, will the process be the same?

I didn't intend this new section to cover what happens for a minor or major EE point release but I'm not 100% sure if you are referring to the TCK or SPEC or EE versioning?

Reply:
@kwsutter I just pushed an update to address this by referring to service releases.